### PR TITLE
Fix origin nomenclature sft (center/corner)

### PR DESCRIPTION
**Pull request by frheault**
Fix the inversion in the documentation about the standard (trackvis vs nifti).
Replace the unclear variable name (`shifted_origin` -> `origin_at_corner`)
Added a small catch for all-zero affine that was crashing in `nib.aff2axcodes`

### CODE REVIEW

**Comment by frheault** [comment]
@Garyfallidis I hope it does not impact your ongoing visualisation projects, the variable has the same behavior as before (a find&replace should do the trick).

File: dipy/io/utils.py [hunk 1]
```diff
@@ -192,6 +192,10 @@ def get_reference_info(reference):
         affine[2, 0:4] = header['srow_z']
         dimensions = header['dim'][1:4]
         voxel_sizes = header['pixdim'][1:4]
+
+        if not affine.any():
```
Comment (arokem) [inline dipy/io/utils.py]: Could you please add a test for this behavior?

**Comment by skoudoro** [comment]
Thank you for this update @frheault. I just want to merge #1965 before your PR otherwise it looks good.

**Comment by arokem** [comment]
The appveyor error is unrelated. I think this is ready to go -- any objections?

**Comment by GuillaumeTh** [comment]
@arokem Code looks good. This PR could be merged now to allow @frheault to continue #2013. It could be nice to have #2013 in next release.

**Comment by skoudoro** [comment]
No objection, you can go ahead @arokem

**Comment by arokem** [comment]
Off we go. Thanks @frheault !

**Comment by arokem** [comment]
We should really figure out how to turn off that pesky appveyor...

**Comment by skoudoro** [comment]
agree

**Comment by arokem** [comment]
I think maybe I did that now. For now, I just tried checking as many items in the settings not to build, but it would also be possible to delete the project from appveyor altogether. I am a bit reluctant to do that, because it would mean that we would lose all the history of builds.

**Comment by skoudoro** [comment]
I tried just tried to remove the Appveyor webhook and service on Github and it seems that a new one is created... strange...

**Pull request merged**